css.js

```diff
@@ -0,0 +1 @@
+require('./dist/lib/css');
```

Comment:
It doesn't really make a difference whether we expose it through `module.exports` or not, though, as far as I can tell.